ingeniamotion/wizard_tests/stoppable.py

```diff
@@ -24,16 +24,21 @@ def wrapper(self, *args, **kwargs):  # type: ignore
         return wrapper
 
     def reset_stop(self) -> None:
-        self.is_stopped = False
+        if self.stop_queue.full():
+            self.stop_queue.get(block=False)
```

Comment:
Same race condition, just deleted the .full() condition

Reply:
@polfeliu Done [fe1299d](https://github.com/ingeniamc/ingeniamotion/pull/177/commits/fe1299dffcffa1751c4b9862a4b49d88d2512d25)